.wwhrd.yml

```diff
@@ -0,0 +1,7 @@
+allowlist:
```

Comment:
Can we rename the file to have `yaml` extension rather than `yml`, as we already have files with the 2nd extesion

Reply:
done.